concepts/05 Widgets/Scheduler/030 Appointments/015 Appointment Types/030 Recurring Appointments.md

```diff
@@ -1,40 +1,34 @@
-A recurring appointment is an appointment that repeats after a specified period of time. 
+A recurring appointment is an appointment that repeats at a certain interval. 
 
 ![Scheduler Recurring Appointment](/images/UiWidgets/Scheduler_RecurrentAppointment.png)
 
-An object defining a recurring appointment should contain the [recurrenceRule](/api-reference/50%20Common/Object%20Structures/dxSchedulerAppointment/recurrenceRule.md '/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule') field that follows the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification.
+To make an appointment recurring in code, specify the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) field (it follows the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification). You can exclude certain appointments from the created series. Specify the [recurrenceException](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceException) field to set their start date and time.
```

Comment:
- "(it follows the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification)" - avoid putting information in brackets if possible because it is usually skipped when reading. 
- "from the created series" - the term "series" has not been introduced yet.

```suggestion
To make an appointment recurring, specify the [recurrenceRule](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceRule) field with a value according to the <a href="https://tools.ietf.org/html/rfc2445#section-4.3.10" target="_blank">iCalendar RFC 2445</a> specification. This creates an appointment series. You can exclude specific appointments from it if you set their [recurrenceException](/Documentation/ApiReference/Common/Object_Structures/dxSchedulerAppointment/#recurrenceException) field. For example, the appointment below occurs daily since February 20, 2021, except February 22 and 23:
```